test/fixtures/custom-outputs/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "custom-outputs",
+  "main": "dist/custom-outputs.js",
```

Comment:
This test doesn't actually test `"main"` correctly, as setting main creates a confusing mess of filenames (see #825 & #896).

`"main": "dist/custom-outputs.cjsx.js"` would make (say) the `module` output file path into `dist/custom-outputs.cjsx.modulex.mjs`. That is the "correct" output name at this time, but would be confusing as hell to debug as it doesn't match the `package.json`, so I've skipped it.